src/test/java/eu/europeana/metis/sandbox/service/workflow/TransformationServiceImplTest.java

```diff
@@ -77,7 +77,7 @@ void transform_invalidXml_expectFail() throws IOException, TransformationExcepti
         .thenThrow(new TransformationException(new Exception("Failing here")));
 
     RecordProcessingException exception = assertThrows(RecordProcessingException.class,
-        () -> service.transform(record));
+        () -> service.transformToEdmInternal(record));
```

Comment:
Comment not related to this line.
We need to test the new method in this test class as well right?

Reply:
Added new unit tests for the new method